e3/vcs/svn.py

```diff
@@ -107,7 +107,11 @@ def current_revision(self):
         :rtype: str
         :raise: SVNError
         """
-        return self.get_info('Last Changed Rev')
+        try:
+            return self.get_info('Last Changed Rev')
+        except Exception as e:
+            logger.exception("Cannot fetch last changed rev")
+            raise SVNError("Cannot fetch last changed rev", "svn_cmd")
```

Comment:
Can we include the exception message as well? It might contain some information helping diagnose the issue...